Vostok.Logging.File/Rolling/Strategies/HybridRollingStrategy.cs

```diff
@@ -13,27 +13,30 @@ internal class HybridRollingStrategy : IRollingStrategy
         private readonly Func<DateTime> timeProvider;
         private readonly IFileSuffixFormatter<DateTime> timeSuffixFormatter;
         private readonly IFileSuffixFormatter<(DateTime, int)> hybridSuffixFormatter;
+        private readonly char suffixEliminator;
```

Comment:
Why isn't it "hot"? Would prefer a Func<char> here so that config changes can be applied without app restart. This comment also applies to other strategies (TimeBased, SizeBased), as well as HybridSuffixFormatter.